### PR TITLE
add osx_arm64 build

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -53,3 +53,4 @@ extra:
     - hmaarrfk
     # - jakirkham
     # - dougalsutherland
```
Comment (hmaarrfk) [inline recipe/meta.yaml]: might as well remove the lines on top.

**Comment by hmaarrfk** [comment]
Why not add this to the migrator? I feel like some c hack is missing and the migrator would get it for you?

Also why remove ncurses?

**Comment by matthiasdiener** [comment]
> Why not add this to the migrator? I feel like some c hack is missing and the migrator would get it for you?

Sure, we could do that, although I believe it is failing currently because osx_arm64 libssh hadn't propagated yet to the mirrors

> Also why remove ncurses?

ncurses has run_exports now, so I guess it is not necessary here anymore?

**Comment by matthiasdiener** [comment]
@conda-forge-admin restart ci

**Comment by hmaarrfk** [comment]
I see. Looks great!

**Comment by matthiasdiener** [comment]
@conda-forge-admin, please rerender

**Comment by matthiasdiener** [comment]
This is ready for review/merge @hmaarrfk

**Pull request merged**